homeassistant/components/netatmo/button.py

```diff
@@ -0,0 +1,76 @@
+"""Support for Netatmo/Bubendorff button."""
+
+from __future__ import annotations
+
+import logging
+
+from pyatmo import modules as NaModules
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_URL_CONTROL, NETATMO_CREATE_BUTTON
+from .data_handler import HOME, SIGNAL_NAME, NetatmoDevice
+from .entity import NetatmoModuleEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Netatmo button platform."""
+
+    @callback
+    def _create_entity(netatmo_device: NetatmoDevice) -> None:
+        entity = NetatmoCoverPreferredPositionButton(netatmo_device)
+        _LOGGER.debug("Adding button %s", entity)
+        async_add_entities([entity])
+
+    entry.async_on_unload(
+        async_dispatcher_connect(hass, NETATMO_CREATE_BUTTON, _create_entity)
+    )
+
+
+class NetatmoCoverPreferredPositionButton(NetatmoModuleEntity, ButtonEntity):
+    """Representation of a Netatmo cover preferred position button device."""
+
+    _attr_configuration_url = CONF_URL_CONTROL
+    device: NaModules.Shutter
+
+    def __init__(self, netatmo_device: NetatmoDevice) -> None:
+        """Initialize the Netatmo device."""
+        super().__init__(netatmo_device)
+
+        self.entity_description = ButtonEntityDescription(
```

Comment:
we should use `entity_description` when we want to have a generic entity class that handles multiple entities based on their EntityDescriptions. In this case we don't have that, so you can just set `_attr_translation_key` and `_attr_entity_registry_enabled_default` directly as class attributes